deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/GeneratedOpenApiFile.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkiverse.openapi.generator.deployment;
+
+import org.jboss.jandex.ClassInfo;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public class GeneratedOpenApiFile extends MultiBuildItem {
```

Comment:
Classes extending BuildItems should always be final. If this is meant to be reused, make it abstract